src/widget/wsearchlineedit.h

```diff
@@ -74,12 +80,15 @@ class WSearchLineEdit : public QComboBox, public WBaseWidget {
     // value provider that sends signals whenever the corresponding
     // configuration value changes.
     static int s_debouncingTimeoutMillis;
+    static bool s_searchCompletionsEnabled;
+    static bool s_searchHistoryShortcutsEnabled;
```

Comment:
```suggestion
    static bool s_searchCompletionsEnabled;
    static bool s_searchHistoryShortcutsEnabled;
```
just because without 'search' prefix readability in the cpp is better IMO, and less redundant e.g. when getting it from within the preferences `WSearchLineEdit::kEnableCompletionsByDefault`